tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -102,50 +161,142 @@ Select "Pupil Exclusions" publication
-    user checks table column heading contains  1   1   2020 Week 14
+    # 1 YEAR AFTER STUDY
+    user checks results table row heading contains  2  1   British
+    user checks results table row heading contains  3  1   Mixed/Multiple ethnic group
+    user checks results table row heading contains  4  1   Not Known/Not Provided
+    user checks results table row heading contains  5  1   Other Ethnic Group
+    user checks results table row heading contains  6  1   Total
+    user checks results table row heading contains  7  1   White
+    # 2 YEAR AFTER STUDY
+    user checks results table row heading contains  8  2   Asian/Asian British
+    user checks results table row heading contains  9  1   Black/African/Caribbean/Black British
+    user checks results table row heading contains  10  1  Mixed/Multiple ethnic group
+    user checks results table row heading contains  11  1  Not Known/Not Provided
+    user checks results table row heading contains  11  1  Not Known/Not Provided
+    user checks results table row heading contains  12  1  Other Ethnic Group
+    user checks results table row heading contains  13  1  Total
+    user checks results table row heading contains  14  1  White
+    # 3 YEAR AFTER STUDY
+    user checks results table row heading contains  15  2  Asian/Asian British
+    user checks results table row heading contains  16  1  Black/African/Caribbean/Black British
+    user checks results table row heading contains  17  1  Mixed/Multiple ethnic group
+    user checks results table row heading contains  18  1  Not Known/Not Provided
+    user checks results table row heading contains  19  1  Other Ethnic Group
+    user checks results table row heading contains  20  1  Total
+    user checks results table row heading contains  21  1  White
+    # 4 YEAR AFTER STUDY
+    user checks results table row heading contains  22  2  Asian/Asian British
+    user checks results table row heading contains  23  1  Black/African/Caribbean/Black British
+    user checks results table row heading contains  24  1  Mixed/Multiple ethnic group
+    user checks results table row heading contains  25  1  Not Known/Not Provided
+    user checks results table row heading contains  26  1  Other Ethnic Group
+    user checks results table row heading contains  27  1  Total
+    user checks results table row heading contains  28  1  White
+    # 5 YEAR AFTER STUDY
+    user checks results table row heading contains  29  2  Asian/Asian British
+    user checks results table row heading contains  30  1  Black/African/Caribbean/Black British
+    user checks results table row heading contains  31  1  Mixed/Multiple ethnic group
+    user checks results table row heading contains  32  1  Not Known/Not Provided
+    user checks results table row heading contains  33  1  Other Ethnic Group
+    user checks results table row heading contains  34  1  Total
+    user checks results table row heading contains  35  1  White
+
+Validate table cells
```

Comment:
How big is this table? I'm a little concerned that it's too big to reasonably write all the assertions for and we're leaving ourselves open for potential bugs due to not covering the entire table.

I would consider cutting back the selections to something where we can test the entire table properly